cstrike/addons/amxmodx/scripting/map_manager_effects.sma

```diff
@@ -45,10 +45,12 @@ new bool:g_bFreezeFlagsChanged;
 new HamHook:g_hHamSpawn;
 
 new const g_sSound[][] = {
-    "sound/fvox/one.wav", "sound/fvox/two.wav", "sound/fvox/three.wav", "sound/fvox/four.wav", "sound/fvox/five.wav",
-    "sound/fvox/six.wav", "sound/fvox/seven.wav", "sound/fvox/eight.wav", "sound/fvox/nine.wav", "sound/fvox/ten.wav"
+    "fvox/one.wav", "fvox/two.wav", "fvox/three.wav", "fvox/four.wav", "fvox/five.wav",
+    "fvox/six.wav", "fvox/seven.wav", "fvox/eight.wav", "fvox/nine.wav", "fvox/ten.wav"
 };
 
+new g_sStartSound[] = "Gman/Gman_Choose2.wav";
```

Comment:
new _const_

Reply:
https://github.com/Mistrick/MapManagerModular/blob/master/cstrike/addons/amxmodx/scripting/include/map_manager_stocks.inc#L71

Here in stock this arg is not const so it will be error on compiling